### PR TITLE
Update changelog for 6.14.2

### PR DESCRIPTION
**Pull request by albertvaka**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+
+Release on: 2019-10-29
+
+
+.. _Release Notes_6.14.2_Bug Fixes:
+
+Bug Fixes
+---------
+
+- In Windows, allows the install to succeed successfully even in the event
```
Comment (olivielpeau) [inline CHANGELOG.rst]:
```suggestion
- On Windows, allows the install to succeed successfully even in the event
```

(I think?)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+  that the user was not cleaned up successfully from a previous install.
+
+
```
Comment (olivielpeau) [inline CHANGELOG.rst]: one change to add IMO:

```suggestion
- On Windows, do not remove the home folder of the Agent's user (`dd-agent-user`) on uninstall.

```

**Review by olivielpeau — COMMENTED** [review]
A couple of comments

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+6.14.2
+======
+
+.. _Release Notes_6.14.2_Prelude:
+
+Prelude
+-------
+
+Release on: 2019-10-29
```
Comment (KSerrania) [inline CHANGELOG.rst]: We should mention in the prelude that this is a Windows-only release.

**Review by KSerrania — APPROVED** [review]

**Review by dsahni — APPROVED** [review]
lgtm

**Review by olivielpeau — APPROVED** [review]

**Pull request merged**